src/app/containers/ArticleTimestamp/index.stories.jsx

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
+import timemachine from 'timemachine';
 import ArticleTimestamp from '.';
 import { timestampGenerator } from './testHelpers';
 import { ServiceContextProvider } from '../../contexts/ServiceContext';
 import services from '../../lib/config/services';
 
+timemachine.config({
+  dateString: 'Tue, 01 Jan 2019 00:00:00 GMT',
+  timestamp: '1546300800',
+});
+
```

Comment:
This overrides the Date object so that it uses the configuration here, to generate the `const`s below.